Geometry/MTDNumberingBuilder/plugins/DDCmsMTDConstruction.cc

```diff
@@ -1,5 +1,3 @@
-//#define EDM_ML_DEBUG
```

Comment:
Do you want to remove also this comment of debug? Because I still see `#ifdef EDM_ML_DEBUG` in the code.

Reply:
I would guess there is nothing left to test, next time it may be added.

Reply:
Not sure I understand. I see, i.e. 
```
#ifdef EDM_ML_DEBUG
    edm::LogVerbatim("MTDNumbering") << "Module = " << fv.name() << " fullNode = " << fullNode
                                     << " thisNode = " << thisNode;
#endif
```
in the code. It is not a big deal for me. I just want to confirm if this is by intention.

Reply:
Just to note that the preferred way is to define the `EDM_ML_DEBUG` macro at the compilation command
```
USER_CXXFLAGS="-DEDM_ML_DEBUG" scram b
```
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger#LogDebug

(that doesn't provide file-level control, but the message categories are the intended ones for finer-grained control)

Reply:
I very often use file-level control, and in the past I was used to add commented ```#define EDM_ML_DEBUG``` statements in many places. But since the file needs anyway to be edited, adding the statement directly when needed does not really make any difference, and the code looks cleaner.